Dockerfile

```diff
@@ -6,7 +6,7 @@ WORKDIR /src
 
 RUN npm install
 
-RUN npm run build
+RUN npm run build-dev
```

Comment:
Use the Dockerfile-development file, don't change Dockerfile for the dev environment